lib/beaker/dsl/helpers/host_helpers.rb

```diff
@@ -497,7 +497,7 @@ def file_exists_on(host, file_path)
             if file_path.include?(':')
               split_path = win_ads_path(file_path)
 
-              command = %(Test-Path #{split_path[:path]})
+              command = %(Test-Path "#{split_path[:path]}")
```

Comment:
Is this to support paths with spaces in them? 
I think you would need it here too and on line 495 above for the case where there is no ads path?

https://github.com/puppetlabs/beaker/pull/1647/files#diff-cdd2e965e7b543eb19634118d43851b8R495

Reply:
It is to protect windows path (c:\asdasd) from cygwin shell expansion. Without it `C:\Windows\TEMP\install-puppet-2020-05-15_15.38.14.log` gets modified to `C:\cygwin64\home\Administrator\WindowsTEMPinstall-puppet-2020-05-15_15.38.14.log`